client/blueprints/task/files/lib/__name__/app/serializers/__name__.js

```diff
@@ -0,0 +1,2 @@
+import TaskSerializer from 'tahi/serializers/task';
+export default TaskSerializer.extend();
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(end)' and instead saw 'default'.